README.rst

```diff
@@ -21,9 +21,9 @@ supported Python versions.
 +--------------------------+--------------------------+---------------------------------+
 | ``torch``                | ``torchvision``          | ``python``                      |
 +==========================+==========================+=================================+
-| ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.7``, ``<=3.10``           |
+| ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.7.2``, ``<=3.10``         |
 +--------------------------+--------------------------+---------------------------------+
-| ``1.13.0``               | ``0.14.0``               | ``>=3.7``, ``<=3.10``           |
+| ``1.13.0``               | ``0.14.0``               | ``>=3.7.2``, ``<=3.10``         |
```

Comment:
Thanks @datumbox . Are we planning on a bugfix release soon? Otherwise it might be more accurate to leave this line untouched (and update the `X` in `0.14.X`)

Reply:
I am not aware of any bugfix releases any time soon. I think we should update the table to reflect that 3.7.0 won't work.